File: app/src/main/java/se/hactar/movieregister/viewmodel/MovieListViewModel.java
package se.hactar.movieregister.viewmodel;


import android.arch.lifecycle.LiveData;
import android.arch.lifecycle.ViewModel;

import java.util.List;

import se.hactar.movieregister.MovieApp;
import se.hactar.movieregister.model.Movie;

public class MovieListViewModel extends ViewModel {

    public LiveData<List<Movie>> getLiveDataMovies() {
        return MovieApp.getDb().movieDao().getAllLiveData();
    }
}

File: app/src/main/java/se/hactar/movieregister/viewmodel/MovieListViewModel.kt
package se.hactar.movieregister.viewmodel


import androidx.lifecycle.LiveData
import androidx.lifecycle.ViewModel
import se.hactar.movieregister.MovieApp
import se.hactar.movieregister.model.Movie

class MovieListViewModel : ViewModel() {

    fun liveDataMovies(): LiveData<List<Movie>> {
        return MovieApp.database.movieDao().allLiveData
    }
}
